src/main/java/com/minecolonies/util/BlockUtils.java

```diff
@@ -112,6 +111,13 @@ public static boolean isWater(IBlockState iBlockState)
                || Objects.equals(iBlockState, Blocks.flowing_water.getDefaultState());
     }
 
+
+    public static boolean isBlockSeed(World world, BlockPos pos)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)